resources/definitions/dxu2.def.json

```diff
@@ -0,0 +1,90 @@
+{
+    "version": 2,
+    "name": "UM2+ DXUv2 Single",
+    "inherits": "ultimaker2_plus",
+    "metadata":
+    {
+        "visible": true,
+        "author": "TheUltimakerCommunity",
+        "manufacturer": "DXU",
+        "file_formats": "text/x-gcode",
+        "platform": "ultimaker2_platform.obj",
+        "has_machine_materials": false,
+        "has_machine_quality": false,
+        "has_materials": true,
+        "has_variant_materials": false,
+        "has_variants": true,
+        "machine_extruder_trains":
+        {
+            "0": "dxu2_extruder1",
```

Comment:
These two files are not included.

Reply:
@nallath I have add the missing extruder files. Thanks.